doors_dashboards/components/meteogram.py

```diff
@@ -70,22 +81,25 @@ def _get_wrapped_meteogram_image(
             lon, lat, time, meteogram_type
         )
         return dbc.Col(
-            id=sub_component_id,
-            children=[
-                meteogram_image,
-            ],
-            style={
+            meteogram_image,
+            # width="50%",
+            style={  # 'marginBottom': '15px',
+                'color': FONT_COLOR,
+                'fontSize': FONT_SIZE,
+                'fontFamily':
+                    FONT_FAMILY,
+                'fontWeight': 'bold',
+                'backgroundColor': PLOT_BGCOLOR,
+                'padding': '35px',
+                'border-radius': '15px',
                 'flex': '1',
-                'margin': '-34px 0 0 0',
+                # 'margin-top': '4px',
```

Comment:
```suggestion
```

Whenever something is commented out, leave a comment why it is not deleted. In most cases, it is something that should be deleted.